src/js/components/Navigation/MainHeaderBar.jsx

```diff
@@ -98,7 +98,7 @@ export default function MainHeaderBar () {
             <Suspense fallback={<span>ZzZzZzZ</span>}>
               <SignInButton classes={classes} />
             </Suspense>
-            <div>
+            <div style={{ padding: '8px 0 0 0' }}>
```

Comment:
I think we should strive to avoid direct style statements like this, and instead use a classes reference like `className={classes.dropDownWrapper}`. And if in a component file, a styled component.